### PR TITLE
Government Overhaul

### PR DESCRIPTION
**Pull request by loup99**
Overhauled governments in the mod:
- Added four new government types, including Sub-Roman, High Tribal, Gupta and Pre-Feudal
- Added events for a Sub-Roman to Feudal transition, with the abandoning of Roman taxation, the social status of the army, crisis caused by war and changing cultural values of the aristocracy
- Added and adapted decisions and laws for a Tribal to High Tribal to Feudal government transitions, with appropriate factions, casus bellis and plots
- Removed the feudal government from the map at start
- Added the possibility for Sub-Roman rulers to become Bureaucratic
- Overhauled government types attributed in history
- Fixed the duchy of Deheubarth not being present

### CODE REVIEW

File: WTWSMS/history/titles/d_deheubarth.txt [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+}
+1219.5.4={
+	liege = k_england
+	holder = 7348 #William Marshal
+}
+1272.11.16={
+	holder = 454520
+}
+1307.7.8={
+	holder = 454510 #Edward II of England
```
Comment (IhateTrains) [inline WTWSMS/history/titles/d_deheubarth.txt]: WtWSMS doesn't have this character, presumably some older ones too.

File: WTWSMS/events/WtWSMS_government_events.txt [hunk 2]
```diff
@@ -0,0 +1,961 @@
...
+	only_playable = yes
+	
+	trigger = {
+		liege = {
+			primary_title = {
+				has_title_flag = sub_roman_government_flag
+			}
+			government = sub_roman_government
+		}
+		NOT = { set_character_flag = learning_value_vassal } # Don't want the event to fire again
```
Comment (IhateTrains) [inline WTWSMS/events/WtWSMS_government_events.txt]:
```suggestion
		NOT = { has_character_flag = learning_value_vassal } # Don't want the event to fire again
```

File: WTWSMS/events/WtWSMS_government_events.txt [hunk 3]
```diff
@@ -0,0 +1,961 @@
...
+	only_playable = yes
+	
+	trigger = {
+		liege = {
+			primary_title = {
+				has_title_flag = sub_roman_government_flag
+			}
+			government = sub_roman_government
+		}
+		NOT = { set_character_flag = military_value_vassal } # Don't want the event to fire again
```
Comment (IhateTrains) [inline WTWSMS/events/WtWSMS_government_events.txt]:
```suggestion
		NOT = { has_character_flag = military_value_vassal } # Don't want the event to fire again
```

File: WTWSMS/history/titles/d_deheubarth.txt (hunk 1 above)
Comment (loup99) [reply WTWSMS/history/titles/d_deheubarth.txt]: It was a copy of the one of a county, but in the meantime the one of the county had changed. Fixed now.

File: WTWSMS/events/WtWSMS_government_events.txt (hunk 2 above)
Comment (loup99) [reply WTWSMS/events/WtWSMS_government_events.txt]: An oversight that has been fixed.

**Comment by IhateTrains** [comment]
> --- Error 1 of 1 ---
> At <mod>\events\WtWSMS_government_events.txt [character_event\option\custom_tooltip\text] (Line 932, column 4):
> "WtWSMS_govs.16.a.tooltip" is not a valid LocalizationKey.
>

**Comment by loup99** [comment]
> WtWSMS_govs.16.a.tooltip

This oversight should be fixed now.

**Pull request merged**